vue/test/components/dropdown/dropdown.test.js

```diff
@@ -0,0 +1,142 @@
+const assert = require("assert");
+const base = require("../../base");
+
+describe("Dropdown", () => {
+    it("should show a message if there are no items defined", () => {
+        const items = [];
+        const messageEmpty = "No items";
+        const component = base.getComponent("Dropdown", {
+            props: { items: items, messageEmpty: messageEmpty }
+        });
+
+        const dropdownItems = component.findAll(".dropdown-item").length;
+        const messageEmptyItem = component.findAll(".dropdown-item-empty").length;
+
+        assert.strictEqual(dropdownItems, 0);
+        assert.strictEqual(messageEmptyItem, 1);
+    });
+
+    it("should have items accordingly with props", () => {
+        const items = [
+            {
+                value: "text_1",
+                label: "Text 1"
+            },
+            {
+                value: "text_2"
+            }
+        ];
+        const component = base.getComponent("Dropdown", {
+            props: { items: items }
+        });
+        const itemsLength = component.findAll(".dropdown-item").length;
+        const firstItem = component.findAll(".dropdown-item").at(0);
+        const secondItem = component.findAll(".dropdown-item").at(1);
+        assert.strictEqual(itemsLength, 2);
+        assert.strictEqual(firstItem.text(), "Text 1");
+        assert.strictEqual(secondItem.text(), "text_2");
+        assert.strictEqual(component.vm.$props.items, items);
+    });
+
+    it("should close on item click", async () => {
+        const items = [
+            {
+                value: "text_1",
+                label: "Text 1"
+            },
+            {
+                value: "text_2"
+            }
+        ];
+        const component = base.getComponent("Dropdown", {
+            props: { items: items }
+        });
+
+        const item = component.findAll(".dropdown-item").at(0);
+        await item.trigger("click");
+        assert.strictEqual(component.vm.$data.visibleData, false);

```

Comment:
Check if if the component is openend to ensure the click is the one that changed `visibleData` state